docs/release/generate_release_notes.py

```diff
@@ -78,25 +98,59 @@ def tqdm(i, **kwargs):
 else:
     raise RuntimeError(f'Desired tag ({args.from_commit}) not found')
 
+common_ancestor = local_repo.merge_base(args.to_commit, args.from_commit)[0]
+remote_commit = repository.get_commit(common_ancestor.hexsha)
+
 # For some reason, go get the github commit from the commit to get
 # the correct date
```

Comment:
Is this implementation comment still relevant now?

```suggestion
```

Reply:
I still get commit from remote, not tagged one, but the common ancestor of tag and branch.

Reply:
Yeah, the reason seems more clear after the new code and variable names you added. So this comment doesn't seem to add much now. But I'd let you decide if you want to keep it.